From 5571d5df2356464bfa815031bafa6dd47229d76e Mon Sep 17 00:00:00 2001
From: Lina Chen <chenln1124@thundersoft.com>
Date: Fri, 6 Jan 2023 08:34:08 +0000
Subject: [PATCH 4/8] Fix utils and machine static library install.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index da246ee0d..c0997b18a 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -400,7 +400,7 @@ install(TARGETS kvsWebrtcClient
 )
 endif()
 
-install(TARGETS kvsWebrtcSignalingClient
+install(TARGETS kvsWebrtcSignalingClient kvsWebrtcUtils kvsWebrtcStateMachine
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
 )
-- 
2.17.1

