From 5b31af789b0684f5f39594fea9300727ebe1411c Mon Sep 17 00:00:00 2001
From: Lina Chen <chenln1124@thundersoft.com>
Date: Fri, 6 Jan 2023 02:29:04 +0000
Subject: [PATCH 4/4] Fix ld error.

---
 CMakeLists.txt | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ec4657e..ff7ea73 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -27,12 +27,12 @@ if(WSLAY_TESTS)
 	add_subdirectory(tests)
 endif()
 
-if (WSLAY_CONFIGURE_INSTALL)
-	include(GNUInstallDirs)
-	set(INSTALL_CMAKE_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/wslay)
-	install(TARGETS wslay
-		DESTINATION ${INSTALL_CMAKE_DIR})
-	configure_file(wslay-config.cmake.in wslay-config.cmake @ONLY)
-	install(FILES ${CMAKE_CURRENT_BINARY_DIR}/wslay-config.cmake
-		DESTINATION ${INSTALL_CMAKE_DIR})
-endif()
+# if (WSLAY_CONFIGURE_INSTALL)
+# 	include(GNUInstallDirs)
+# 	set(INSTALL_CMAKE_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/wslay)
+# 	install(TARGETS wslay
+# 		DESTINATION ${INSTALL_CMAKE_DIR})
+	# configure_file(wslay-config.cmake.in wslay-config.cmake @ONLY)
+	# install(FILES ${CMAKE_CURRENT_BINARY_DIR}/wslay-config.cmake
+	# 	DESTINATION ${INSTALL_CMAKE_DIR})
+# endif()
-- 
2.17.1

